recipes-openxt/tss/tpm2-tss/leave-autotools-to-oe.patch

```diff
@@ -0,0 +1,16 @@
+--- a/bootstrap
++++ b/bootstrap
+@@ -13,7 +13,6 @@ src_listvar () {
+ }
+ 
+ VARS_FILE=src_vars.mk
+-AUTORECONF=${AUTORECONF:-autoreconf}
```

Comment:
Instead of patching the file, we could call `AUTORECONF=true ./bootstrap` from do_configure_prepend() to skip executing autoreconf.

Reply:
Indeed! It is preferable.